Casks/insomnia.rb

```diff
@@ -9,8 +9,14 @@
   homepage "https://insomnia.rest/"
 
   livecheck do
-    url :url
-    regex(/^core@?(\d+(?:\.\d+)+)$/i)
+    url "https://insomnia.rest/changelog"
+    strategy :page_match do |page|
+      latest_release = page[/"release_version"\s*:\s*"(\d+(?:\.\d+)*)"/i, 1]
+      update_info = Homebrew::Livecheck::Strategy.page_content("https://updates.insomnia.rest/builds/check/mac?v=#{latest_release}&app=com.insomnia.app&channel=stable")[:content]
+      next latest_release if update_info.blank?
+
+      update_info.scan(%r{/Insomnia.Core-(\d+(?:\.\d+)*)\.zip}i).flatten
```

Comment:
@bevanjkay What do you think, should I go with:
```suggestion
      update_info[/"name"\s*:\s*"(\d+(?:\.\d+)*)"/i, 1]
```